ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4835,6 +4846,33 @@ DX2DSPRTC:
   - CONTROL
 
 
+HPDSPRTC:
```

Comment:
This looks the same as DX2DSPRTC. What is the justification for creating a separate type for the heat pump?